src/Evolution/Executables/GeneralizedHarmonic/CMakeLists.txt

```diff
@@ -5,13 +5,24 @@ function(add_generalized_harmonic_executable
     INITIAL_DATA_NAME INITIAL_DATA BOUNDARY_CONDITIONS LIBS_TO_LINK)
   add_spectre_parallel_executable(
     "EvolveGh${INITIAL_DATA_NAME}"
-    EvolveGeneralizedHarmonic
+    EvolveGeneralizedHarmonicWithHorizon
     Evolution/Executables/GeneralizedHarmonic
     "EvolutionMetavars<${INITIAL_DATA}, ${BOUNDARY_CONDITIONS}>"
     "${LIBS_TO_LINK}"
     )
 endfunction(add_generalized_harmonic_executable)
 
+function(add_generalized_harmonic_no_horizon_executable
+    INITIAL_DATA_NAME INITIAL_DATA BOUNDARY_CONDITIONS LIBS_TO_LINK)
+  add_spectre_parallel_executable(
+    "EvolveGh${INITIAL_DATA_NAME}"
+    EvolveGeneralizedHarmonic
+    Evolution/Executables/GeneralizedHarmonic
+    "EvolutionMetavars<${INITIAL_DATA}, ${BOUNDARY_CONDITIONS}>"
+    "${LIBS_TO_LINK}"
```

Comment:
Worth it to modify this so as not to unnecessarily link ApparentHorizons here? (ApparentHorizons library is in LIBS_TO_LINK)